WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -309,12 +309,18 @@ extension UIImage {
         return UIImage(named: "login-site-address-info")!
     }
 
-    /// Login error: no jetpack
+    /// Login error: no Jetpack
     ///
     static var loginNoJetpackError: UIImage {
         return UIImage(named: "woo-no-jetpack-error")!
     }
 
+    /// Login error: no WordPress
+    ///
+    static var loginNoWordPressError: UIImage {
+        return UIImage(named: "woo-wp-no-site")!
```

Comment:
I think this has to be rotated for RTL layouts right?
```suggestion
        return UIImage(named: "woo-wp-no-site")!.imageFlippedForRightToLeftLayoutDirection()
```